### ISSUE TITLE
Monster's active conditions are not saved in savegame file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#253]
```
Confirmed in v0.6.10a3.

1) Equip the Chaosreaper
2) Attack a monster and make sure that the condition is applied (visible by the 
change in monster stats)
3) Exit to menu.
4) Continue game.
5) View the monster stats again.
6) Notice how the condition has vanished.

This is due to the fact that conditions on monsters are not stored in the 
savegame file. Instead, monster conditions should be persisted on a per-monster 
basis, so that active monster conditions are remembered when restoring a 
previous savegame.
```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 12 Oct 2011 at 9:59

**Comment by GoogleCodeExporter** [comment]
```
Should be resolved in r177.
```

Original comment by `oskar.wi...@gmail.com` on 12 Oct 2011 at 10:06
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
